canonical-web-and-design/ubuntu.com
# WIP - updated /kubernetes/features page

## Done

- Updating copy as per [copydoc](https://docs.google.com/document/d/1mEOBW4pfsWJqTLm9Pn9myKtaslD-QB2RvUKcNQ52bXU/edit#)
- Updated styling as per [redesign](https://github.com/canonical-web-and-design/web-squad/issues/4461)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Check against above copydoc and redesign and ensure all changes have been made correctly 
**NOTE: This is still a WIP as we are waiting on some assets**

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4462

## Screenshots


![Screenshot 2021-10-01 at 12 37 47](https://user-images.githubusercontent.com/17607612/135606835-b22d9aa8-acc1-458e-ab12-2708d5b68bbc.png)

![Screenshot 2021-10-01 at 12 38 26](https://user-images.githubusercontent.com/17607612/135606850-abca038e-d870-4ff9-bc9d-084150a208eb.png)
